src/fides/api/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -119,6 +141,64 @@ def _filter_experiences_by_region_or_country(
     return db.query(PrivacyExperience).filter(False)
 
 
+@router.get(
+    urls.PRIVACY_EXPERIENCE_META,
+    status_code=HTTP_200_OK,
+    response_model=Page[PrivacyExperienceMetaResponse],
+)
+@fides_limiter.limit(CONFIG.security.public_request_rate_limit)
+async def get_privacy_experience_meta(
+    *,
+    db: Session = Depends(deps.get_db),
+    params: Params = Depends(),
+    region: Optional[str] = None,
+    component: Optional[ComponentType] = None,
+    request: Request,  # required for rate limiting
+    response: Response,  # required for rate limiting
+) -> AbstractPage[PrivacyExperienceMetaResponse]:
+    """Minimal Developer Friendly Privacy Experience endpoint that returns only the meta object,
+    the component, and the region."""
+
+    logger.info("Fetching meta info for Experiences '{}'", params)
+
+    await asyncio.sleep(delay=0.001)
+    experience_query: Query = db.query(PrivacyExperience)
+
+    await asyncio.sleep(delay=0.001)
+    if region is not None:
+        experience_query = _filter_experiences_by_region_or_country(
+            db=db, region=region, experience_query=experience_query
+        )
+
+    await asyncio.sleep(delay=0.001)
+    if component is not None:
+        experience_query = _filter_experiences_by_component(component, experience_query)
+
+    await asyncio.sleep(delay=0.001)
+    # TCF contents are the same across all EEA regions, so we can build this once.
+    base_tcf_contents: TCFExperienceContents = get_tcf_contents(db)
+
+    tcf_meta: Optional[Dict] = None
+    for (
+        tcf_section_name,
+        _,
+    ) in TCF_SECTION_MAPPING.items():
+        if getattr(base_tcf_contents, tcf_section_name):
+            # TCF Experience meta is also the same across all EEA regions. Only build meta if there is TCF
+            # content under at least one section.
+            tcf_meta = build_experience_tcf_meta(base_tcf_contents)
+            break
+
+    results: List[PrivacyExperience] = []
+    for experience in experience_query:
+        if experience.component == ComponentType.tcf_overlay:
+            # Attach meta for TCF Experiences only.  We don't yet build meta info for non-TCF experiences.
+            experience.meta = tcf_meta
+        results.append(experience)
+
+    return fastapi_paginate(results, params=params)
```

Comment:
at first glance, it struck me as a bit odd that we're paging and wrapping this within a `PrivacyExperience`, but the doc was helpful in understanding rationale, and i think it makes sense!